script-library/docker-debian.sh

```diff
@@ -124,6 +127,18 @@ fi
 . /etc/os-release
 # Fetch host/container arch.
 architecture="$(dpkg --print-architecture)"
+target_compose_arch="${architecture}"
```

Comment:
Yeah this situation is a bit odd because certain things want what dpkg --print-architecture returns (e.g. amd64 or arm64) while others want a subset of what uname provides. (e.g. aarch64 -- though uname can also return amd64).